<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="2" hostname="sysadm-Latitude-3480" name="paymentsUI.PaymentUI_Progress" tests="4" failures="1" timestamp="05 Mar 2020 05:12:34 GMT" time="1.783" errors="1">
  <testcase name="afterMethod" time="0.001" classname="paymentsUI.PaymentUI_Progress">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at org.openqa.selenium.remote.Augmenter.extractRemoteWebDriver(Augmenter.java:72)
at org.openqa.selenium.remote.BaseAugmenter.augment(BaseAugmenter.java:96)
at common.CommonUtil.screenshot(CommonUtil.java:179)
at common.CommonUtil.afterMethod(CommonUtil.java:142)
at paymentsUI.PaymentUI_Progress.afterMethod(PaymentUI_Progress.java:82)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:458)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:646)
at org.testng.internal.Invoker.retryFailed(Invoker.java:839)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1010)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
    </error>
  </testcase> <!-- afterMethod -->
  <testcase name="firePaymentURLandValidatePaymentModes" time="0.976" classname="paymentsUI.PaymentUI_Progress">
    <failure type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError
at org.junit.Assert.fail(Assert.java:92)
at org.junit.Assert.assertTrue(Assert.java:43)
at org.junit.Assert.assertTrue(Assert.java:54)
at paymentsUI.PaymentUI_Progress.firePaymentURLandValidatePaymentModes(PaymentUI_Progress.java:58)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.retryFailed(Invoker.java:839)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1010)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
    </failure>
  </testcase> <!-- firePaymentURLandValidatePaymentModes -->
  <testcase name="firePaymentURLandValidatePaymentModes" time="0.806" classname="paymentsUI.PaymentUI_Progress">
    <skipped/>
  </testcase> <!-- firePaymentURLandValidatePaymentModes -->
  <testcase name="inProgressPayment" time="0.000" classname="paymentsUI.PaymentUI_Progress">
    <skipped/>
  </testcase> <!-- inProgressPayment -->
</testsuite> <!-- paymentsUI.PaymentUI_Progress -->
